### PR TITLE
docs: Added the parent locator (..)

### LINKED ISSUE #16243: Please add the .. (parent) locator to the docs
> You know you can get to the parent by doing
> ```
> page.locator("#locator").locator("..").locator("div").nth(0)
> ```

_Originally posted by @sgreener in https://github.com/microsoft/playwright/issues/15234#issuecomment-1170212273_

### PR DESCRIPTION
**Pull request by 304NotModified**
Fixes #16243

### CODE REVIEW

File: docs/src/selectors.md [hunk 1]
```diff
@@ -1167,6 +1167,32 @@ document
 
 If a selector needs to include `>>` in the body, it should be escaped inside a string to not be confused with chaining separator, e.g. `text="some >> text"`.
 
+## Parent selector
+
+The parent could be selected with `..`.
```
Comment (dgozman) [inline docs/src/selectors.md:1172]:
```suggestion
The parent could be selected with `..`, which is a short form for `xpath=..`.
```

File: docs/src/selectors.md [hunk 2]
```diff
@@ -1167,6 +1167,32 @@ document
 
 If a selector needs to include `>>` in the body, it should be escaped inside a string to not be confused with chaining separator, e.g. `text="some >> text"`.
 
+## Parent selector
```
Comment (dgozman) [inline docs/src/selectors.md:1170]: I think this now breaks "Intermediate matches" being a subitem of "Chaining selectors". I'd put this whole section right above the "Chaining selectors".

Comment (304NotModified) [reply docs/src/selectors.md:1170]: Ow yes indeed, thx! Will fix this

File: docs/src/selectors.md [hunk 3]
```diff
@@ -1167,6 +1167,32 @@ document
...
+```js
+const parentLocator = elementLocator.locator('..');
+```
+
+```java
+Locator parentLocator = elementLocator.locator("..");
+```
+
+```python async
+parentLocator = elementLocator.locator('..')
```
Comment (mxschmitt) [inline docs/src/selectors.md:1185]:
```suggestion
parent_locator = element_locator.locator('..')
```

File: docs/src/selectors.md [hunk 4]
```diff
@@ -1167,6 +1167,32 @@ document
...
+```
+
+```python sync
+parentLocator = elementLocator.locator('..')
```
Comment (mxschmitt) [inline docs/src/selectors.md:1189]:
```suggestion
parent_lcator = element_locator.locator('..')
```

File: docs/src/selectors.md (hunk 2 above)
Comment (304NotModified) [reply docs/src/selectors.md:1151]: Done!

**Review by dgozman — APPROVED** [review]
Thank you for the PR!

**Pull request merged**